Tools/CISupport/ews-build/steps.py

```diff
@@ -959,54 +1051,105 @@ def skip_build(self, reason):
         bug_id = self.getProperty('bug_id', '') or self.get_bug_id_from_patch(patch_id)
 
         bug_closed = self._is_bug_closed(bug_id) if self.verifyBugClosed else 0
         if bug_closed == 1:
             self.skip_build('Bug {} is already closed'.format(bug_id))
-            return None
+            return False
 
         obsolete = self._is_patch_obsolete(patch_id) if self.verifyObsolete else 0
         if obsolete == 1:
             self.skip_build('Patch {} is obsolete'.format(patch_id))
-            return None
+            return False
 
         review_denied = self._is_patch_review_denied(patch_id) if self.verifyReviewDenied else 0
         if review_denied == 1:
             self.skip_build('Patch {} is marked r-'.format(patch_id))
-            return None
+            return False
 
         cq_plus = self._is_patch_cq_plus(patch_id) if self.verifycqplus else 1
         if cq_plus != 1:
             self.skip_build('Patch {} is not marked cq+.'.format(patch_id))
-            return None
+            return False
 
         acceptable_review_flag = self._does_patch_have_acceptable_review_flag(patch_id) if self.verifycqplus else 1
         if acceptable_review_flag != 1:
             self.skip_build('Patch {} does not have acceptable review flag.'.format(patch_id))
-            return None
+            return False
 
         if obsolete == -1 or review_denied == -1 or bug_closed == -1:
             self.finished(WARNINGS)
-            return None
+            return False
+        return True
 
-        if self.verifyBugClosed:
-            self._addToLog('stdio', 'Bug is open.\n')
-        if self.verifyObsolete:
-            self._addToLog('stdio', 'Patch is not obsolete.\n')
-        if self.verifyReviewDenied:
-            self._addToLog('stdio', 'Patch is not marked r-.\n')
-        if self.verifycqplus:
-            self._addToLog('stdio', 'Patch is marked cq+.\n')
-            self._addToLog('stdio', 'Patch have acceptable review flag.\n')
-        self.finished(SUCCESS)
-        return None
+    def validate_github(self, pr_number):
+        if not pr_number:
+            return False
+
+        repository_url = self.getProperty('repository', '')
+        title = self.getProperty('github.title', '')
+        owners = self.getProperty('owners', [])
+
+        if self.addURLs and title and repository_url:
+            self.addURL('[PR {}] {}'.format(pr_number, title), GitHub.pr_url(pr_number, repository_url))
+        if self.addURLs and owners:
+            contributors, errors = Contributors.load(use_network=False)
+            for error in errors:
+                print(error)
+                self._addToLog('stdio', error)
+            github_username = owners[0]
+            contributor = contributors.get(github_username, {})
+            display_name = contributor.get('email', github_username)
+            self.addURL('Change by {}'.format(display_name), '{}/{}'.format(GITHUB_URL, github_username))
```

Comment:
I think "PR by:" would be better to display instead of "Change by"

Also GITHUB_URL already have a slash in it (at the end), can remove the extra '/' here.

Reply:
For exactly what to display here (username/full-name/email), let's decide based on feedback (waiting for more feedback). But it wouldn't be wise to block this PR on that feedback. Let's land either way, and change it based on feedback later on.

Reply:
Easy enough to change in any case